modules/meeting/spec/features/recurring_meetings/recurring_meeting_crud_spec.rb

```diff
@@ -90,7 +90,7 @@
 
     show_page.delete_meeting_series
     show_page.within_modal "Delete meeting series" do
-      page.check "I understand that this deletion cannot be reversed"
+      check "I understand that this deletion cannot be reversed", allow_label_click: true
```

Comment:
If all check boxes will need this, maybe it would be better to enable it globally with [`automatic_label_click`](https://www.rubydoc.info/gems/capybara/3.12.0/Capybara#configure-class_method)
```ruby
Capybara.configure do |config|
  config.automatic_label_click = true
end
```

Reply:
@cbliard yes, I was thinking we could try enabling globally.